### PR TITLE
add P2 project proposal

### PR DESCRIPTION
**Pull request by angelagongli**

### CODE REVIEW

File: P2-Proposals/TeamPi/P2Proposal.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+6. Generate ML prediction with Python
+7. Email user with case count data, graphs, predictions
+
+## Tech Stack
+* UiPath
+    * Excel activities
+    * Outlook activities
+* Python
+
+## Team Pi
```
Comment (kaurrevature) [inline P2-Proposals/TeamPi/P2Proposal.md]: Please embed the flowchart in this file only.

**Review by kaurrevature — COMMENTED** [review]
I see the flowchart is incomplete, please define data processing and other workflows

**Comment by mariellenolasco** [comment]
Creating your own ML might take some time. Is there an existing one that you're going to leverage?

**Comment by mariellenolasco** [comment]
Furthermore, the flowchart needs some more detail on what the data processing stage entails.

File: P2-Proposals/TeamPi/P2Proposal.md (hunk 1 above)
Comment (angelagongli) [reply P2-Proposals/TeamPi/P2Proposal.md]: We will describe data processing in detail, I will embed the flowchart in our project proposal file!

**Comment by angelagongli** [comment]
> Creating your own ML might take some time. Is there an existing one that you're going to leverage?

The ML idea is Mann's, he is trying to implement k-fold clustering

**Comment by angelagongli** [comment]
> Furthermore, the flowchart needs some more detail on what the data processing stage entails.

We will update the flowchart once we have finished describing it in detail

**Comment by angelagongli** [comment]
I have updated our project proposal with our updated flowchart. Mann has indicated that ML predictions are something we would want to come back to, but not to implement right away.

**Comment by angelagongli** [comment]
I have fixed the embedded flowchart in our project proposal

**Review by mariellenolasco — CHANGES_REQUESTED** [review]
Fix the flowchart, make sure you ask for the user email at the top as we discussed

**Comment by angelagongli** [comment]
I have updated the project proposal flowchart

**Comment by kaurrevature** [comment]
LGTM, good job on the idea

**Pull request merged**